packages/platform/platform-fastify/src/interfaces/interfaces.ts

```
@@ -0,0 +1,21 @@
+import {Env} from "@tsed/core";
+
+import {PlatformFastifySettings} from "./PlatformFastifySettings.js";
+
+export * from "./PlatformFastifySettings.js";
+
+export type PlatformFastifyPluginTypes = Function | /*Type<any> |*/ string;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)